docs/sdk/wallets/intro.md

```diff
@@ -19,6 +19,8 @@ Ready to jump right in? Quickly learn how you can integrate Programmable Key Pai
 
 ## Overview
 
+Each PKP is functionally a wallet, where the private key lives across the Lit Network. It is represented as an ERC-721 NFT, and the owner of the NFT becomes the designated “controller” of the Programmable Key Pair. The controller has the ability to assign additional signing logic and [authentication](../authentication/overview) mechanisms to their PKP using Lit Actions.
```

Comment:
can we use 'authentication methods' instead of 'mechanisms'

Reply:
Done @a1ttech :)